.github/workflows/test-conda.yml

```diff
@@ -0,0 +1,27 @@
+name: Python Package using Conda
+
+on: [push]
+
+jobs:
+  build-linux:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 5
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v3
+      with:
+        python-version: '3.10'
+    - name: Add conda to system path
+      run: |
+        # $CONDA is an environment variable pointing to the root of the miniconda directory
+        echo $CONDA/bin >> $GITHUB_PATH
+    - name: Install dependencies
+      run: |
+        conda env update --file environment.yml --name base
+    - name: Test with tox
+      run: |
+        pip install tox
```

Comment:
Given that your `tox.ini` requires tox-conda I think you will need to `pip install tox-conda` here.

I found that tox-conda was useful for SOLikeT as it has a lot of dependencies which are fussy about compilers, fftw versions etc and tox-conda allows you to manage those kind of system-level tools well.

For something which is more python only, or doesn't build anything I think using the standard tox would be fine (and probably more robust).